### ISSUE TITLE
[Bug] "Use email instead" crashes fenix in Romanian language

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/fenix#9698]
## Steps to reproduce
Set default device language to Romanian
1. Open Fenix
2. Click on "Autentificare în Firefox" (Sign in to Firefox)
3. Click on "Folosește e-mail-ul în schimb" (Use email instead)

### Expected behavior
4. Redirect to "Enter your email to continue to Firefox Sync" page

### Actual behavior
4. Application crashes: "Firefox Preview has stopped"

Note:
This feature works fine using English and Italian languages

### Device information

* Android device: Galaxy S8, Android 8.0.0
* Fenix version: 4.2.1 (Build #20941431)

```
04-03 14:20:59.445   634   634 E hw-IPCThreadState: binder thread pool (1 threads) starved for 108 ms
04-03 14:21:05.829   634   634 E hw-IPCThreadState: binder thread pool (1 threads) starved for 118 ms
04-03 14:21:12.177   634   634 E hw-IPCThreadState: binder thread pool (1 threads) starved for 114 ms
04-03 14:21:15.408   634   634 E hw-IPCThreadState: binder thread pool (1 threads) starved for 102 ms
04-03 14:21:27.681 27669 27669 E AndroidRuntime: FATAL EXCEPTION: main
04-03 14:21:27.681 27669 27669 E AndroidRuntime: Process: org.mozilla.fenix, PID: 27669
04-03 14:21:27.681 27669 27669 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.mozilla.fenix/org.mozilla.fenix.settings.account.AuthCustomTabActivity}: java.util.UnknownFormatConversionException: Conversion = '&'
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2955)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3030)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread.-wrap11(Unknown Source:0)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1696)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:105)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6938)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: Caused by: java.util.UnknownFormatConversionException: Conversion = '&'
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter$FormatSpecifier.conversion(Formatter.java:2781)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2811)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter$FormatSpecifierParser.<init>(Formatter.java:2624)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter.parse(Formatter.java:2557)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter.format(Formatter.java:2504)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.util.Formatter.format(Formatter.java:2458)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at java.lang.String.format(String.java:2814)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.content.res.Resources.getString(Resources.java:472)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.content.Context.getString(Context.java:572)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at org.mozilla.fenix.customtabs.CustomTabToolbarMenu.<init>(CustomTabToolbarMenu.kt:21)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at org.mozilla.fenix.components.toolbar.BrowserToolbarView.<init>(BrowserToolbarView.kt:72)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at org.mozilla.fenix.browser.BaseBrowserFragment.initializeUI(BaseBrowserFragment.kt:57)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at org.mozilla.fenix.customtabs.ExternalAppBrowserFragment.initializeUI(ExternalAppBrowserFragment.kt:1)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at org.mozilla.fenix.browser.BaseBrowserFragment.onViewCreated(BaseBrowserFragment.kt:1)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:94)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.addAddedFragments(FragmentManager.java:5)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:54)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:11)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:4)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:5)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:9)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:99)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:4)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:218)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:9)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:1)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1340)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.Activity.performStart(Activity.java:7200)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2918)
04-03 14:21:27.681 27669 27669 E AndroidRuntime: 	... 9 more
04-03 14:21:27.775  6225  6225 E Zygote  : isWhitelistProcess - Process is Whitelisted
04-03 14:21:27.782  3356  3440 E PhoneWindow_APM : : sendNotificationForAppPermissionMonitor callingPid = 3356, uid = 1000
04-03 14:21:27.782  3356  3440 E PhoneWindow_APM : : sendNotificationForAppPermissionMonitor 2
04-03 14:21:27.782  3356  3440 E PhoneWindow_APM : : sendNotificationForAppPermissionMonitor 21000_3356
04-03 14:21:27.782  3356  3440 E PhoneWindow_APM : : isCalledPackage return false
04-03 14:21:27.782  3356  3440 E PhoneWindow_APM : : sendNotificationForAppPermissionMonitor mIsMonitoredFeature = true
04-03 14:21:27.844  6242  6242 E Zygote  : isWhitelistProcess - Process is Whitelisted
04-03 14:21:27.870  3053  3053 E audit   : type=1400 audit(1585948887.852:32244): ```

**Comment by mcarare** [comment]
I've checked the strings in 4.2.1 and there seems to be a problem with the text from translation. It has been corrected since, but still, we should add a way to handle this in the future.

**Comment by sflorean** [comment]
Tested also with Francais, Espaniol, Deutsch, one RTL language and Chinese, and couldn't reproduce the crash, on 4.2.1 Build.
On Nightly, 04/07 the issue is not reproducible with the Romanian language set.

**Comment by sflorean** [comment]
Verified as fixed on 4.3.0-beta.1 with Samsung Galaxy Note 10 (Android 10) and Nokia 6 (Android 7.1.1). Checked as well on Nightly (the issue wasn't reproducible) and the crash is not encountered.
